roles/testnode/tasks/zypper_systems.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Tasks common to all systems that use the zypper package manager
+# This is mostly a copy of the yum_systems.yml
+
+- name: Set mode on /etc/fuse.conf
+  file:
+    path: /etc/fuse.conf
+    mode: 0644
+    state: touch
+  changed_when: false
+
+- name: Ensure the group kvm exists.
+  group:
+    name: kvm
+    state: present
+
+- name: Add the teuthology user to groups kvm,disk
+  user:
+    name: "{{ teuthology_user }}"
+    groups: kvm,disk
+    append: yes
+
+- name: Configure /etc/sudoers.
+  template:
+    src: sudoers
+    dest: /etc/sudoers
+    owner: root
+    group: root
+    mode: 0440
+    validate: visudo -cf %s
+  tags:
+    - sudoers
+
+- name: Configure /etc/security/limits.conf
+  template:
+    src: limits.conf
+    dest: /etc/security/limits.conf
+    group: root
+    owner: root
+    mode: 0644
```

Comment:
If the only difference in the code was the mode, 0755 in the generic code looks like a bug